### PR TITLE
Small cleanup of monitors package.

### PR DESCRIPTION
**Pull request by Donnerbart**
Just a minor cleanup without changes in the logic.

There was one unused memory gauge which should be used or removed.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java [hunk 1]
```diff
@@ -381,6 +380,8 @@ private void renderHeap() {
                     .append(numberToUnit(runtimeUsedMemory.read())).append(", ");
             sb.append("heap.memory.free=")
                     .append(numberToUnit(runtimeFreeMemory.read())).append(", ");
+            sb.append("heap.memory.available=")
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java]: `runtimeAvailableMemory` was unused, so either we add it here or we should remove it.

File: hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java [hunk 2]
```diff
@@ -498,19 +499,19 @@ private void renderOperationService() {
...
      * @return a string of the given number as a format float with two decimal places and a period
      */
-    public static String percentageString(double p) {
+    private static String percentageString(double p) {
         return format("%.2f%%", p);
     }
 
-    public static String numberToUnit(long number) {
-        //CHECKSTYLE:OFF
+    @SuppressWarnings("checkstyle:magicnumber")
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java]: No need for a complete suppression on those few lines :)

File: hazelcast/src/main/java/com/hazelcast/internal/monitors/InvocationPlugin.java [hunk 3]
```diff
@@ -45,17 +45,15 @@
      *
      * If set to 0, the plugin is disabled.
      */
-    public static final HazelcastProperty SAMPLE_PERIOD_SECONDS
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/internal/monitors/InvocationPlugin.java]: Should stay public.

**Comment by pveentjer** [comment]
All the properties should remain public so I can easily make use of them.

**Comment by Donnerbart** [comment]
None is used outside the package, but I can revert that part. IntelliJ 2016 really got a bit messy with that visibility check enabled by default (even though I switched it to a weak warning) :(

**Comment by pveentjer** [comment]
:+1: 

merge once green

**Pull request merged**